show/muxcable.py

```diff
@@ -14,15 +15,15 @@
 
 REDIS_TIMEOUT_MSECS = 0
 
-CONFIG_SUCCESSFUL = 101
+CONFIG_SUCCESSFUL = 0
 CONFIG_FAIL = 1
 EXIT_FAIL = 1
 EXIT_SUCCESS = 0
 STATUS_FAIL = 1
-STATUS_SUCCESSFUL = 102
+STATUS_SUCCESSFUL = 0
 
 VENDOR_NAME = "Credo"
-VENDOR_MODEL = "CAC125321P2PA0MS"
+VENDOR_MODEL_REGEX = re.compile("CAC.+321P2P.+MS")
```

Comment:
Maybe we can tighten this regex up a bit, like `"CAC\w{3}321P2P\w{2}MS"`?

Reply:
fixed